force-app/main/userCommunity/classes/HOT_WorkOrderListController.cls

```diff
@@ -112,15 +112,21 @@ public without sharing class HOT_WorkOrderListController {
                 ORDER BY StartDate
             ]
         );
+        DateTime oneWeekAgo = DateTime.now().addDays(-7);
+        DateTime sixMonthsAgo = DateTime.now().addMonths(-6);
+
         Map<Id, WorkOrder> workOrdersAccount = new Map<Id, WorkOrder>();
         Map<Id, WorkOrder> workOrdersOrderer = new Map<Id, WorkOrder>();
         for (WorkOrder wo : workOrders.values()) {
             if (wo.HOT_Request__r.Orderer__c == user.AccountId) {
                 wo.HOT_Request__r.Orderer__r.Name = null;
             }
+            if (wo.EndDate < oneWeekAgo) {
+                wo.HOT_Request__r.ActualUserName__c = null;
```

Comment:
Tolkbrukers navn er et annet felt: UserName__c.
Tenkte på om vi bør putte den under  `if (wo.HOT_Request__r.Orderer__c == user.AccountId)`, men blir jo kun satt på bestilling for bruker, så skal ikke være nødvendig.